policy/org.freedesktop.fwupd.policy.in

```diff
@@ -194,4 +194,16 @@
       <allow_active>auth_admin_keep</allow_active>
     </defaults>
   </action>
+
+  <action id="org.freedesktop.fwupd.set-bios-attribute">
+    <description>Set a BIOS attribute</description>
+    <!-- TRANSLATORS: this is the PolicyKit modal dialog -->
+    <message>Authentication is required to write BIOS attributes</message>
+    <defaults>
+      <allow_any>auth_admin</allow_any>
+      <allow_inactive>no</allow_inactive>
+      <allow_active>auth_admin_keep</allow_active>
```

Comment:
We probably want auth_admin -- i.e. ask for each one we set rather than just trust the app for this permission until closed.